### PR TITLE
[DUOS-2549][risk=no] Replace everit json with networknt

### PR DESCRIPTION
**Pull request by rushtong**
### Addresses
https://broadworkbench.atlassian.net/browse/DUOS-2549

### Summary
Minor refactoring PR that replaces an unsupported library with a supported one.

----
Have you read [CONTRIBUTING.md](../CONTRIBUTING.md) lately? If not, do that first.

- Label PR with a Jira ticket number and include a link to the ticket
- Label PR with a security risk modifier [no, low, medium, high]
- PR describes scope of changes
- Get a minimum of one thumbs worth of review, preferably two if enough team members are available
- Get PO sign-off for all non-trivial UI or workflow changes
- Verify all tests go green
- Test this change deployed correctly and works on dev environment after deployment

### CODE REVIEW

**Review by fboulnois — APPROVED** [review]
Looks like there are some failing tests that need to be fixed, but this change looks good!

File: src/main/resources/data-use-v3.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-  "id": "https://consent-ontology.dsde-prod.broadinstitute.org/schemas/data-use",
+  "$id": "https://consent-ontology.dsde-prod.broadinstitute.org/schemas/data-use",
```
Comment (fboulnois) [inline src/main/resources/data-use-v3.json:2]: good catch

**Review by fboulnois — APPROVED** [review]
👍

File: src/main/resources/data-use-v3.json (hunk 1 above)
Comment (rushtong) [reply src/main/resources/data-use-v3.json:2]: Thank you, SO led me to https://github.com/networknt/json-schema-validator/issues/590 for this one

**Review by kaitlynoflaherty — APPROVED** [review]
Looks good!

**Pull request merged**